how-to/common/public/views/manager-portal/common/common.js

```diff
@@ -15,7 +15,49 @@ export async function loadData(filename) {
 }
 
 export async function loadTeamData() {
-	return loadData('team.json');
+	const teamData = loadData('team.json');
+	return await teamData
```

Comment:
Can we change this pattern to remove `.then` there shouldn't be a need to do that.

So

```
const teamData = await loadData('team.json`);

const updatedTeamMembers = [];
for (const member of teamData) {
// update code
}

return updatedTeamMembers .
```

Reply:
Fixed!